lession9_js/main.js

```diff
@@ -0,0 +1,176 @@
+/*
+Bài 1
+Viết một chương trình nhập lương nhân viên, tính thuế thu nhập và lương ròng
+ (số tiền lương thực sự mà nhân viên đó nhận được). 
+ Với các thông số giả sử như sau (không theo luật lương, 
+ chỉ là con số giả sử để dễ tính toán):
+•	30% thuế thu nhập nếu lương là 15 triệu.
+•	20% thuế thu nhập nếu lương từ 7 đến 15 triệu.
+•	10% thuế thu nhập nếu lương dưới 7 triệu.
+ */
+const salary = (s) => {
+  let t 
+  if (s > 15*Math.pow(10,6)) {
+    t = 0.3
+  }
+  else if (s >= 7*Math.pow(10,6) && s <= 15*Math.pow(10,6)) {
+    t = 0.2
+  }
+  else if (s > 0 && s < 7*Math.pow(10,6)) {
```

Comment:
Chú ý cách trình bày
```suggestion
  } else if (s > 0 && s < 7 * Math.pow(10,6)) {
```